### PR TITLE
feat(ui): added helpbub

### PR DESCRIPTION
**Pull request by alinashklyar**

### CODE REVIEW

File: _includes/docs-navbar.html [hunk 1]
```diff
@@ -93,6 +93,19 @@
       </ul>
     </div>
   - <!-- </div>
-    <a class="btn btn-primary order-2 order-md-2" href="https://codefresh.io/docs/" style="background-color: #212437; border-color: #212437; color: #5FFFBF; font-weight: 500">View Codefresh Docs</a> 
+    <a class="btn btn-primary order-2 order-md-2" href="https://codefresh.io/docs/" style="background-color: #212437; border-color: #212437; color: #5FFFBF; font-weight: 500">View Codefresh Docs</a>
   </div> -->
+
+    {% if page.layout != "home" %}
+      <button
+        onclick="window.CommandBar.toggleHelpHub()"
```
Comment (olegt-codefresh) [inline _includes/docs-navbar.html:101]: Can we also add accessibility attributes to this button?

Comment (alinashklyar) [reply _includes/docs-navbar.html:101]: sure, you mean aria-labels?

Comment (alinashklyar) [reply _includes/docs-navbar.html:101]: but it has text inside of it btw

File: _includes/scripts.html [hunk 2]
```diff
@@ -34,19 +34,35 @@
...
+  window.CommandBar.boot(
+    "docs-test",
```
Comment (olegt-codefresh) [inline _includes/scripts.html:57]: Need to check the API add see if we need to rename the `docs-test` (I think it's a field for a user name)

File: _includes/scripts.html [hunk 3]
```diff
@@ -34,19 +34,35 @@
...
...;var r=Symbol.for("CommandBar::configuration"),e=Symbol.for("CommandBar::orgConfig"),c=Symbol.for("CommandBar::disposed"),i=Symbol.for("CommandBar::isProxy"),m=Symbol.for("CommandBar::queue"),l=Symbol.for("CommandBar::unwrap"),d=[],s=localStorage.getItem("commandbar.lc"),u=s&&s.includes("local")?"http://localhost:8000":"https://api.commandbar.com",f=Object.assign(((n={})[r]={uuid:o},n[e]={},n[c]=!1,n[i]=!0,n[m]=new Array,n[l]=function(){return f},n),a.CommandBar),p=["addCommand","boot"],y=f;Object.assign(f,{shareCallbacks:function(){return{}},shareContext:function(){return{}}}),a.CommandBar=new Proxy(f,{get:function(o,n){return n in y?f[n]:p.includes(n)?function(){var o=Array.prototype.slice.call(arguments);return new Promise((function(a,t){o.unshift(n,a,t),f[m].push(o)}))}:function(){var o=Array.prototype.slice.call(arguments);o.unshift(n),f[m].push(o)}}}),null!==s&&d.push("lc=".concat(s)),d.push("version=2"),t("".concat(u,"/latest/").concat(o,"?").concat(d.join("&")),!0)}}void 0===Object.assign||"undefined"==typeof Symbol||void 0===Symbol.for?(a.__CommandBarBootstrap__=r,t("https://polyfill.io/v3/polyfill.min.js?version=3.101.0&callback=__CommandBarBootstrap__&features="+n)):r();
```
Comment (olegt-codefresh) [inline _includes/scripts.html:55]: Please make sure it's the latest code snippet from the commandbar docs:
https://www.commandbar.com/docs/guides/installation/installation-types/installing-in-web-app/#option-2-snippet

File: _includes/scripts.html (hunk 2 above)
Comment (olegt-codefresh) [reply _includes/scripts.html:57]: Let's check with Idan if we should add fixed `id` like `"documentation-user"` or pass an empty string as anonymous user.

image

File: index.html [hunk 4]
```diff
@@ -19,7 +19,17 @@ <h2 class="mb-3 bd-text-white text-uppercase">
             Instantly build, test, and deploy Docker images.<br>
             Realize your true DevOps potential with the premier enterprise Argo software delivery solution.
           </p>
-          <div id="searchbar"></div>
+<!--          <div id="searchbar"></div>-->
+          <button
```
Comment (olegt-codefresh) [inline index.html:23]: accessibility attributes for the button

File: yarn.lock [hunk 5]
```diff
@@ -2,20 +2,29 @@
...
-  dependencies:
-    commander "^2.8.1"
-    convert-source-map "^1.1.0"
-    fs-readdir-recursive "^1.0.0"
-    glob "^7.0.0"
-    lodash "^4.2.0"
-    output-file-sync "^2.0.0"
-    slash "^1.0.0"
-    source-map "^0.5.0"
+"@ampproject/remapping@^2.2.0":
```
Comment (olegt-codefresh) [inline yarn.lock:5]: Is this change caused by `yarn upgrade` or we have uncommitted changes?

File: _includes/docs-navbar.html (hunk 1 above)
Comment (olegt-codefresh) [reply _includes/docs-navbar.html:101]: Just to make sure it's accessibe via keyboard and a screen reader

File: yarn.lock (hunk 5 above)
Comment (alinashklyar) [reply yarn.lock:5]: i ran 'yarn' command, anyway, i'll clear changes from here

**Review by olegt-codefresh — APPROVED** [review]

**Pull request merged**